ui/src/global.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module '*.jpg' {
```

Comment:
Haven't seen this before. What's it do?

Reply:
?? Again, not sure. It was there when I started working on this branch.

Reply:
@mschu124 any ideas about this one?

Reply:
Its for getting images to work with typescript
https://github.com/wmonk/create-react-app-typescript/blob/master/template/images.d.ts

Reply:
Oh! Alright then!